etc/profile-a-l/chafa.profile

```diff
@@ -0,0 +1,53 @@
+# Firejail profile for chafa
+# Description: A terminal-based image viewer and image-to-text converter.
+# This file is overwritten after every install/update
+# Persistent local customizations
+include chafa.local
+# Persistent global definitions
+include globals.local
+
+blacklist ${RUNUSER}
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-write-mnt.inc
+
+include whitelist-run-common.inc
+include whitelist-runuser-common.inc
+#include whitelist-usr-share-common.inc
```

Comment:
```suggestion
# Add the following to your chafa.local if you do not need to view images in
# /usr/share.
#include whitelist-usr-share-common.inc
```

Add context to the commented code.

Based on the following comment:

* <https://github.com/netblue30/firejail/pull/5355#discussion_r961399395>